testing/generate.js

```diff
@@ -210,17 +213,16 @@ function main() {
     // console.log('error creating directory');
   }
   
-  // TODO: Support protected applications
   const protectedPath = path.join(basePath, '.retool', 'protected-apps.yaml');
-  if (fs.existsSync(protectedPath)) {
-    console.log('Testing Protected Applications in CI is currently not supported');
-    process.exit(1);
+  if (!fs.existsSync(protectedPath)) {
+    console.log('Protected apps repository is missing ./retool/protected-apps.yaml');
+    process.exit(0);
```

Comment:
@mokshjawa will need to update for Gitlab (file does not exist)

Reply:
https://linear.app/retool/issue/DTC-2230/protected-apps-check-in-ci-will-not-work-for-gitlab